worker/src/handlers/aggregationProcessor/aggregationProcessor.js

```diff
@@ -214,12 +237,34 @@ const getAddPipeline = async ({
     };
   }
 
+  // MIDDLE
+
+  let addToMiddle;
+  if (model.lastRun) {
```

Comment:
Probably easier to use a ternary here instead of the ternary you have later in the code as shown below.

```js
...(addToMiddle ? [addToMiddle] : []),
```
So we can change the code here to the following, replacing the `...` with the code below these lines.
```
const addToMiddle = model.lastRun === undefined ? [] : [{ /* ... */ }]
```
Then we can remove the ternary later, I've added a code suggestion for that.